serverless/serverless
# Allow * in variable string literal defaults

## What did you implement:
closes #5225
closes #4959
closes #4967 (older PR) - this implementation isn't quite the same. It just adds `*` to the regex. Quotes are required for it to work, when quotes are omitted, nothing resolves(see tests).

## How did you implement it:
Added `*` to the variable syntax regex

## How can we verify it:
Create a service with `${opt:foobar, "*"}` in a variable and run `sls print` to check that the variable is correctly substituted.

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
